### PR TITLE
moonpay: propagate fiat and language to the widget 

### PR DESCRIPTION
**Pull request by x1ddos**
Preferences are taken straight from the app config.

If a value is unsupported by Moonpay, they fall back to some default
values, according to their documentation. Worst case scenario, users
are still able to change both fiat and language directly in the widget.

### CODE REVIEW

File: frontends/web/test/i18n/config.test.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ describe('language detector', () => {
     it('uses native-locale if userLanguage is null', done => {
         (apiGet as jest.Mock).mockImplementation(endpoint => {
             switch (endpoint) {
-                case 'config': { return Promise.resolve({frontend: {userLanguage: null}}); }
+                case 'config': { return Promise.resolve({backend: {userLanguage: ""}}); }
```
Comment (thisconnect) [inline frontends/web/test/i18n/config.test.tsx:79]: So it used to be stored as null in config.json, how can it be empty string?

Name of the test is 'uses native-locale if userLanguage is null'

Does `migrateUserLanguage` turn null to empty string?

Comment (thisconnect) [reply frontends/web/test/i18n/config.test.tsx:79]: Ok backend conf is now an empty stirng if the lang == native lang 👍 

but the frontend conf is still there, remove?

Screen Shot 2021-01-20 at 10 15 56 AM

**Review by thisconnect — APPROVED** [review]
LGTM with small nit

**Comment by benma** [comment]
The first commit's message states a false premise. You can access the config from the account handler:

`handlers.account.Config()`.

**Review by benma — APPROVED** [review]
utACK

File: frontends/web/test/i18n/config.test.tsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ describe('language detector', () => {
     it('uses native-locale if userLanguage is null', done => {
```
Comment (thisconnect) [inline frontends/web/test/i18n/config.test.tsx:76]: @x1ddos change to `'uses native-locale if userLanguage is an empty string'` ?

Comment (x1ddos) [reply frontends/web/test/i18n/config.test.tsx]: done

File: frontends/web/test/i18n/config.test.tsx (hunk 1 above)
Comment (x1ddos) [reply frontends/web/test/i18n/config.test.tsx:79]: Add this comment. Makes sense?

image

Comment (thisconnect) [reply frontends/web/test/i18n/config.test.tsx:79]: thank you :)

**Pull request merged**